guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ResourceResolver.java

```diff
@@ -331,41 +336,52 @@ public Project resolveToParentProject( final Path resource ) {
     }
 
     @Override
-    public Package resolveParentPackage( final Package pkg ) {
+    public Package resolveDefaultWorkspacePackage(final Project project) {
```

Comment:
@paulovmr @manstis how about formatting the whole project in separate PR using the agreed upon formatting conventions? This way we could avoid having to look for "real" changes in the see of formatting changes.. ;-)